tests/tests.py

```diff
@@ -230,6 +229,7 @@ def test_single_form_csv_with_rep_fields_start_with_repeat(self):
         for line1, line2 in izip(open(get_filename('models.py'),'r'),open(get_filename(cmp_fileName),'r')):
             self.assertEqual(line1,line2)
     def test_single_form_csv_with_rep_fields_many_nested(self):
```

Comment:
this test uses a file that is missing its first row and more from its contents. It cannot be identified as a data dictionary, but without this validation is able to produce output in these tests.

see tests/fixtures/single_form_with_rep_fields_many_nested.csv